App/src/main/res/values/strings.xml

```diff
@@ -88,5 +88,6 @@
     <string name="required">Required.</string>
     <string name="place_name_already_in_use">Name is already in use</string>
     <string name="add_place">Add place</string>
+    <string name="gps_disabled">Your GPS seems to be disabled, do you want to enable it?</string>
```

Comment:
Maybe change this to be a more general location message? Because technically you can get your location without GPS, with only wifi. A message similar as in Google Maps perhaps?
![image](https://cloud.githubusercontent.com/assets/3842311/6226162/26f4d766-b69b-11e4-88e0-6a2cc563f615.png)